test/test_proxy_lib.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+*/
+
+#include <malloc.h>
+
+#include "base.hpp"
+#include "test_helpers.h"
+
+using umf_test::test;
+
+TEST_F(test, proxyLibBasic) {
+
+    ::free(::malloc(64));
+
+#ifndef NDEBUG
```

Comment:
So, right now this is never run since proxy lib only works for Release? Perhaps use some other define for now, until we fix it for Debug?

Reply:
I think we can enable this check in the Release build too.